README.md

```diff
@@ -6,15 +6,14 @@
 [![License](https://img.shields.io/badge/License-BSD--3-blue)](https://github.com/intel/rohd/blob/main/LICENSE)
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](https://github.com/intel/rohd/blob/main/CODE_OF_CONDUCT.md)
 
-
-
-Rapid Open Hardware Development (ROHD) Framework
```

Comment:
The underline style is not allowed by the lint?

Reply:
Rule [MD003](https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md003.md) is triggered.

Reply:
Ah got it, ok, fair enough!